app/models/game.rb

```diff
@@ -2,6 +2,15 @@ class Game < ApplicationRecord
   include Sortable
 
   MAX_PLAYERS_PER_GAME = 8
+  MIN_PLAYERS_PER_GAME = 2
+  VALID_TRANSITIONS = {
+    created: %i[created in_progress canceled],
+    in_progress: %i[in_progress completed canceled],
+    completed: %i[completed],
+    canceled: %i[canceled]
```

Comment:
I've put them because in the model we validate the state on every update, including when we update only the players of the game and then it should let me change the players if I want to.

Reply:
Fixed in my commit